docker-compose.yml

```diff
@@ -133,4 +140,7 @@ services:
       dockerfile: src/Web/WebStatus/Dockerfile
 
   dapr-placement:
-    image: "daprio/dapr:1.9.4"
\ No newline at end of file
+    image: "daprio/dapr:1.12.3"
+    command: ["./placement", "--port", "50006"]
+    ports:
+      - "50006:50006"

```

Comment:
No need to expose those ports 
```suggestion
```

Reply:
agree in fact this collides with Dapr on the host if you have it